nidm/terms/imports/qibo_import.ttl

```diff
@@ -39,35 +39,34 @@ rdf:type rdf:type owl:AnnotationProperty .
 ###  http://www.owl-ontologies.com/Ontology1298855822.owl#Ex_vivo_subject
 
 qibo:Ex_vivo_subject rdf:type owl:Class ;
-                     
+
                      rdfs:label "Ex-vivo Subject"^^xsd:string  .
 
 
 
 ###  http://www.owl-ontologies.com/Ontology1298855822.owl#In_vitro_subject
 
 qibo:In_vitro_subject rdf:type owl:Class ;
-                      
+
                       rdfs:label "In-vitro Subject"^^xsd:string  .
 
 
 
 ###  http://www.owl-ontologies.com/Ontology1298855822.owl#In_vivo_subject
 
 qibo:In_vivo_subject rdf:type owl:Class ;
-                     
+
                      rdfs:label "In-vivo Subject"^^xsd:string  .
 
 
 
 ###  http://www.owl-ontologies.com/Ontology1298855822.owl#Phantom_experimental_subject
 
 qibo:Phantom_experimental_subject rdf:type owl:Class ;
-                                  
+
                                   rdfs:label "Phantom Experimental Subject"^^xsd:string  .
 
 
 
 
 ###  Generated by the OWL API (version 3.5.1) http://owlapi.sourceforge.net
-
```

Comment:
I would have left autogenerated files without changes -- not familiar yet with turtle format intricately but may be those space indentations are important part of the syntax...?  I have tested this one (with trailing spaces removed) at some randomly found http://ttl.summerofcode.be/ -- seems to be ok.  I think we can proceed but might need to be ready to revert this part of cleansing